de.dlr.sc.virsat.graphiti/src/de/dlr/sc/virsat/graphiti/util/DiagramHelper.java

```diff
@@ -88,7 +91,14 @@ public static StructuralElementInstance getOwningStructuralElementInstance(EObje
 			seiUri = seiUri.appendFileExtension(VirSatProjectCommons.FILENAME_EXTENSION);
 			seiUri = URI.createPlatformResourceURI(seiUri.toFileString(), true);
 			
-			Path path = new Path(resourceUri.toPlatformString(false));
+	
+			String decodedPath = null;
+			try {
+				decodedPath = URLDecoder.decode(resourceUri.toPlatformString(false), StandardCharsets.UTF_8.name());
+			} catch (UnsupportedEncodingException e) {
+				e.printStackTrace();

```

Comment:
Same here

Reply:
Added the Activator for logging service here as well